text/01.tex

```diff
@@ -27,165 +30,236 @@ \subsection{Jak na Overleaf?}
-    \item \textbf{Enumerace:} Principielně funguje stejně jako itemizace, použijeme prostředí \texttt{enumerate}  Jednotlivé body jsou pak místo označení pomlček  očíslovány
-    \item U itemizace a enumerace můžeme prostředí do sebe vnořovat
+Nečíslované seznamy se vytváří v~prostředí \verb|itemize|:
+jednotlivé body oddělujeme značkou \verb|\item|.
 
+Číslované seznamy fungují obdobně a~vytváří se prostředím \verb|enumerate|.
+Tato dvě prostředí do~sebe lze vnořovat.
 
-    \begin{multicols}{2}
-    [ITEMIZACE]
-    \setlength{\columnsep}{10cm}
-     \begin{lstlisting}[frame = single]
+\begin{figure}[ht]
+\begin{mdframed}
+\onehalfspacing
+\begin{verbatim}
+\begin{itemize}
+  \item itemize prvni urovne
+  \begin{itemize}
+    \item itemize druhe urovne
+    \begin{enumerate}
+      \item prvni polozka enumerate
+      \item druha polozka enumerate
+    \end{enumerate}
+  \end{itemize}
+\end{itemize}
+\end{verbatim}
+
+\vspace*{1em}
+
+\begin{itemize}
+    \item itemize první úrovně
     \begin{itemize}
-        \item text1
-        \begin{itemize}
-            \item text2
-        \end{itemize}
-    \end{itemize}
-    \end{lstlisting}
-    \columnbreak
-   \begin{centering}
-
-   \end{centering}
-     \begin{itemize}
-        \item text1
-        \begin{itemize}
-            \item text2
-        \end{itemize}
+        \item itemize druhé úrovně
+        \begin{enumerate}
+            \item první položka enumerate
+            \item druhá položka enumerate
+        \end{enumerate}
     \end{itemize}
-    
-    \end{multicols}
-
-          
-        
-        
-    \item Nové odstavce se vytváří automaticky, pokud vynecháme 1 řádek. Řádek jako takový můžeme také zalomit \verb|\\| aniž bychom pak začali nový odstavec viz níže
-    \item Pro nezalomitelnou mezeru můžeme použít \verb|~| -- \verb|v~domě|
 \end{itemize}
+\end{mdframed}
+\end{figure}
+\FloatBarrier
+
+Většina autorů píše stejným způsobem jako ve~Wordu, a~\enquote{Enter} v~kódu píše až na~konci odstavce.
+Psát každou větu na~svůj řádek však přináší výhodu: když je kód verzován pomocí \texttt{git}u, vznikají menší \texttt{diff}y a~změny jsou lépe vidět.
+Tak je strukturován i~tento dokument.
  
-    \textbf{Zalamování řádků}
-    TEXT TEXT TEXT TEXT TEXT\verb|\\|\\ TEXT TEXT TEXT
-    
-    \textbf{Vynechání řádku}
-    TEXTT TEXT TEXT TEXT TEXT \verb|vynechán řádek|
-    
-    TEXT TEXT TEXT TEXT
 \subsection{Vizuální značky}
-\begin{itemize}
-    \item \verb|\textbf{text}| pro tučný text - \textbf{text}
-    \item \verb|\textit{text}| pro kurzívu - \textit{text}
-    \item \verb|\enquote{text}| pro české uvozovky -\enquote{text}
-    \item \verb|\texttt{text}| pro strojový text -\texttt{text}
 
-\end{itemize}
+\begin{table}[ht]
+\centering
+\begin{tabular}{|l|l|}
+značka & ukázka \\
+\hline \hline
+\texttt{textbf} & \textbf{tučné písmo} \\
+\texttt{textit} & \emph{kurzíva} \\
+\texttt{enquote} & \enquote{text v~uvozovkách} \\
+\texttt{texttt} & \texttt{strojový text} \\
```

Comment:
Ještě bych přidal \underline{ }
možná ještě \emph{ }, uplně jsem zapomněl že to existuje dokud jsi to tady nepoužil, i když je to celkem niche věc